### PR TITLE
feat: add user signup feature

### LINKED ISSUE #14: [FEATURE] Create Signup Component for web ui
Is your feature request related to a problem? Please describe.
Create a signing component for the react web app

Describe the solution you'd like
Should use material design consistent with the dev spec
Should follow the design of the created mockup
Should be a stateless component that only displays the form

Additional context
Branch name: ft-signup-web-ui

### PR DESCRIPTION
**Pull request by bekeplar**
## Description
Create the signup component for the web-ui
Closes #14

### CODE REVIEW

**Review by kmwamasali — CHANGES_REQUESTED** [review]
change package.json to remove conflict with main file and resubmit branch

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     },
     "author": {
       "name": "Team 267",
-      "email": "kmwamasali@gmail.com"
+      "email": "bekeplar@gmail.com"
```
Comment (kmwamasali) [inline package.json]: change this back to original

File: src/web_ui/src/components/Forms/Signup.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  input {
+    width: 100%;
+    padding: 6px 20px;
+    margin: 8px 0;
+    box-sizing: border-box;
+  }
+  
+  form {
+    padding-bottom: 5%;
+  }
```
Comment (kmwamasali) [inline src/web_ui/src/components/Forms/Signup.css]: add newline at end of file

**Review by kmwamasali — CHANGES_REQUESTED** [review]
also add this

File: src/web_ui/src/components/Forms/Signup.css (hunk 2 above)
Comment (bekeplar) [reply src/web_ui/src/components/Forms/Signup.css]: Done brother

**Pull request merged**